src/platforms/hosted/cli.c

```diff
@@ -133,76 +134,102 @@ static void cl_help(char **argv)
-		"\t                  1 = INFO, 2 = GDB, 4 = TARGET, 8 = PROBE, 16 = WIRE\n"
+		"\t-h, --help       Show the version and this help, then exit\n"
+		"\t-l, --list       List available supported probes\n"
+		"\t-v, --verbose    <bitmask> Set the output verbosity level based on some combination of:\n"
+		"\t                   1 = INFO, 2 = GDB, 4 = TARGET, 8 = PROBE, 16 = WIRE\n"
 		"\n"
-		"Probe selection arguments [-d PATH | -P NUMBER | -s SERIAL | -c TYPE]:\n"
-		"\t-d <path>       Use a serial device at the given path (Deprecated!)\n"
-		"\t-P <number>     Use the <number>th debug probe found while scanning the\n"
-		"\t                  system, see the output from list for the order\n"
-		"\t-s <serial>     Select the debug probe with the given serial number\n"
-		"\t-c <type>       Select the FTDI-based debug probe with of the given\n"
-		"\t                  type (cable)\n"
+		"Probe selection arguments (If used, choose one):\n"
+		"\t-d, --device     <path> Use a serial device at the given path (Deprecated!)\n"
+		"\t-P, --probe      <number> Use the <number>th debug probe found while scanning the\n"
+		"\t                   system, see the output from list for the order\n"
+		"\t-s, --serial     <serial> Select the debug probe with the given serial number\n"
+		"\t-c, --ftdi-type  <type> Select the FTDI-based debug probe with of the given\n"
+		"\t                   type (cable)\n"
 		"\n"
-		"The default is to start a debug server at localhost:2000\n"
-		"General configuration options: [-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]]\n"
-		"  [-H] [-M STRING ...]\n"
-		"\t-n <number>     Select the target device at the given position in the\n"
-		"\t                  scan chain (use the -t option to get a scan chain listing)\n"
-		"\t-j              Use JTAG instead of SWD\n"
-		"\t-C              Connect to target under hardware reset\n"
-		"\t-t              Perform a chain scan and display information about the\n"
-		"\t                  conected devices\n"
-		"\t-T              Perform continues read- or write-back of a value to allow\n"
-		"\t                  measurement of protocol timing. Aborted by ^C\n"
-		"\t-e              Assume external resistors for FTDI devices, that is having the\n"
-		"\t                  FTDI chip connected through resistors to TMS, TDI and TDO\n"
-		"\t-p              Power the target from the probe (if possible)\n"
-		"\t-R[h]           Reset the device. If followed by 'h', this will be done using\n"
-		"\t                  the hardware reset line instead of over the debug link\n"
-		"\t-H              Do not use the high level command API (bmp-remote)\n"
-		"\t-M <string>     Run target-specific monitor commands. This option\n"
-		"\t                  can be repeated for as many commands you wish to run.\n"
-		"\t                  If the command contains spaces, use quotes around the\n"
-		"\t                  complete command\n"
+		"General configuration options:\n"
```

Comment:
We honestly think that with all section headers, the option summary following is very useful to save a pile of scrolling (good for those who's memory isn't great)